projects/admin/src/assets/rero-ils-ui/admin/i18n/en_US.json

```diff
@@ -196,7 +194,7 @@
   "Limits": "Limits",
   "Loading…": "Loading…",
   "Loan": "Loan",
-  "Loan started at": "Loan started at",
+  "Loan started at": "Loan started",
```

Comment:
Why not *Loan started on* as stated in this weblate comment: https://hosted.weblate.org/translate/rero_plus/rero-ils-admin/fr/?checksum=9ebc20bb300f3158 ?